gradio/components/gallery.py

```diff
@@ -18,6 +19,10 @@
 set_documentation_group("component")
 
 
+GalleryImageType = Union[np.ndarray, _Image.Image, Path, str]
+CaptionedGalleryImageType = Tuple[GalleryImageType, str]
```

Comment:
```suggestion
CaptionedGalleryImageType = tuple[GalleryImageType, str]
```

Reply:
This also seems not to work.
https://github.com/gradio-app/gradio/actions/runs/7151467600/job/19475795051
```
Run . venv/bin/activate
Traceback (most recent call last):
  File "js/_website/generate_jsons/generate.py", line 5, in <module>
    from src import changelog, demos, docs, guides
  File "/home/runner/work/gradio/gradio/js/_website/generate_jsons/src/docs/__init__.py", line 6, in <module>
    import gradio
  File "/home/runner/work/gradio/gradio/gradio/__init__.py", line 3, in <module>
    import gradio._simple_templates
  File "/home/runner/work/gradio/gradio/gradio/_simple_templates/__init__.py", line 1, in <module>
    from .simpledropdown import SimpleDropdown
  File "/home/runner/work/gradio/gradio/gradio/_simple_templates/simpledropdown.py", line 6, in <module>
    from gradio.components.base import FormComponent
  File "/home/runner/work/gradio/gradio/gradio/components/__init__.py", line 27, in <module>
    from gradio.components.gallery import Gallery
  File "/home/runner/work/gradio/gradio/gradio/components/gallery.py", line 23, in <module>
    CaptionedGalleryImageType = tuple[GalleryImageType, str]
TypeError: 'type' object is not subscriptable
```